src/background/utils/requests.js

```diff
@@ -68,6 +71,11 @@ function xhrCallbackWrapper(req) {
   };
 }
 
+function isSpecialHeader(lowerHeader) {
+  return specialHeaders.includes(lowerHeader)
+    || specialHeaderPrefixes.includes(lowerHeader.replace(/-.*/, '-'));
```

Comment:
How about using `startsWith` instead of regular expression since comparison between plain text is much lighter?

Reply:
Fixed. The reason I did it that way was to enable multiple prefixes in theory via `specialHeaderPrefixes`, but you're right, it could just as easily be added as an extra line to the function.